tools/tasks/seed/build.js.test.ts

```diff
@@ -5,7 +5,7 @@ import { join} from 'path';
 
 import { APP_DEST, APP_SRC, TOOLS_DIR, ENABLE_SCSS } from '../../config';
 import { makeTsProject } from '../../utils';
-import { ViewBrokerService } from '../../../src/client/app/frameworks/core/services/view-broker.service';
+import {ViewBroker} from '../../../nativescript/app/app/frameworks/core/utils/view-broker';
```

Comment:
This should stay from `src/client`... nothing in the build should ever import from nativescript app... at least for now.